docs/reference/contributing/guidelines/workflow.md

```diff
@@ -219,11 +219,29 @@ We use many other labels to summarize the scope and effect of the changes.
 
 - *needs: preceding PR* - This pull request cannot yet be merged because it has a dependency on another pull request that needs to merge first.
 - *DO NOT MERGE* - This pull request contains changes that may be in a draft state and submitted purely for review, or may contain breaking changes that have not been considered.
-- *devices: 'name'* - The pull request specifically affects the named device(s).
-- *component: 'name'* - The pull request specifically affects the named component. Component names follow the structure of Mbed OS (`ble`, `storage`, `tls` and so on).
+- *devices: 'name'* - This pull request specifically affects the named device(s).
+- *component: 'name'* - This pull request specifically affects the named component. Component names follow the structure of Mbed OS (`ble`, `storage`, `tls` and so on).
+- *rollup PR* - This pull request is ready for CI testing, but will instead be brought into a rollup pull request to test multiple pull requests at once.
 
 The following labels summarize the scope of the pull request.
 
 - *scope: bug-fix*.
 - *scope: feature*.
 - *scope: new-target*.
+
+#### Rollup pull requests
+
+When Mbed OS has many small, orthogonal pull requests waiting for CI testing to start, maintainers have the option of putting multiple pull requests through CI by creating a rollup pull request. Rollup pull requests are generated by merging other pull requests into a temporary branch, and opening a new pull requests against this new branch. Once the rollup pull request passes CI testing and is merged, the original pull requests that were used to build the rollup pull requests are also automatically merged.
+
+When a pull request is selected to be integrated into a rollup pull requests, it already has a release label and is patiently waiting to start CI testing. Once selected to be looped into a rollup pull request, the original pull request will gain a *rollup PR* label. When the rollup pull request is generated, a comment will also be added to the original pull request mentioning that no further work needs to be done to it. 
```

Comment:
looped?

Reply:
I was trying to come up with a verb for the PRs that get rolled up. Still dunno what to call it.

Reply:
suggestions: rolled-up , combined, integrated, pooled, hooked-up, joined-up, etc... (you can use a thesaurus to find more ideas)

Reply:
added to a rollup pull request?